inst/js/shiny-tracer.js

```diff
@@ -336,30 +351,32 @@ window.shinytest = (function() {
     waitForReady();
 
 
-    $(document).on("shiny:busy", function(e) {
-        shinytest.busy = true;
-        shinytest.log("busy");
-    });
+    function createShinyEventHandlers() {
```

Comment:
JS function hoisting feels so weird

Reply:
Yeah, I've made my peace with it though.